plugins/modules/mount.py

```diff
@@ -734,8 +735,16 @@ def main():
     name = module.params['path']
     changed = False
 
+    lock_tempdir = '/var/tmp'
```

Comment:
Any specific reason to use `/var/tmp` instead of `/tmp`? IIRC `/var/tmp` was useful for temporary files which are needed to persists upon a reboot.

Is it necessary for these lock files to remain after a reboot?

Reply:
no, it isn't. I've changed it.